lib/components/regions_map/components/serach_places/view_models/search_places_bloc.dart

```diff
@@ -38,8 +38,7 @@ class SearchPlacesBloc extends Bloc<SearchPlacesEvent, SearchPlacesState> {
         final res = await GetPlacesAutocompleteUseCase().run(GetPlacesAutocompleteUseCase.input(
           event.query,
           location: Location(lat: position.latitude, lng: position.longitude),
-          language: 'es',
-          components: [Component(Component.country, "mx")],
```

Comment:
:)

Reply:
I think its not really dificult to create a repo for Googe places We are only using the autocomplete end-point (one call)
So we can get the rid of the dependency.